test/integration/puppetlabs/services/jruby/module_info_test.clj

```diff
@@ -0,0 +1,86 @@
+(ns puppetlabs.services.jruby.module-info-test
+  (:require [clojure.test :refer :all]
+            [puppetlabs.kitchensink.core :as ks]
+            [puppetlabs.services.jruby.jruby-puppet-testutils :as jruby-testutils]
+            [puppetlabs.services.jruby.jruby-puppet-service :refer [jruby-puppet-pooled-service]]
+            [puppetlabs.services.puppet-profiler.puppet-profiler-service :refer [puppet-profiler-service]]
+            [puppetlabs.services.jruby-pool-manager.jruby-pool-manager-service :refer [jruby-pool-manager-service]]
+            [puppetlabs.services.jruby.puppet-environments :as puppet-env]
+            [me.raynes.fs :as fs]
+            [cheshire.core :as cheshire]
+            [puppetlabs.puppetserver.testutils :as testutils]
+            [puppetlabs.trapperkeeper.app :as tk-app]
+            [puppetlabs.trapperkeeper.testutils.bootstrap :as tk-bootstrap]))
+
+(defn gen-modules
+  [[mod-dir manifests]]
+  (let [modules-dir (fs/file mod-dir "modules")]
+    (ks/mkdirs! modules-dir)
+    (doseq [manifest manifests]
+      (let [module-dir (fs/file modules-dir manifest)
+            metadata-json {"name" manifest
+                           "version" "1.0.0"
```

Comment:
@camlow325 if you want, I can do what I did with my middleware integration test and make the module version a parameter that can be passed in.